packages/all/project.json

```diff
@@ -64,8 +64,8 @@
       "options": {
         "parallel": false,
         "commands": [
-          "if (test \"${BUILD_BATCH}\" != \"0\") ; then nx run-many --output-style compact -t generate-examples,generate-thumbnails -p ag-charts-website --batch ; fi",
-          "if (test \"${BUILD_BATCH}\" == \"0\") ; then nx run-many --output-style compact -t generate-examples,generate-thumbnails -p ag-charts-website ; fi"
+          "if (test \"${BUILD_BATCH}\" != \"0\") ; then npx nx run-many --output-style compact -t generate-examples,generate-thumbnails -p ag-charts-website --batch ; fi",
```

Comment:
This is kind of weird - why do you need to do this?

Reply:
I was getting a different `nx` version than the repo's version, so this was to rule that out.